src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt

```diff
@@ -238,22 +243,22 @@ class ScriptsTestsIT {
 
     assertEquals(createdScript.id, createdMetaform.scripts!![0])
 
-    val createdScript2 = builder.systemAdmin.scripts.create(script)
-    val updatedMetaform1 = builder.systemAdmin.metaforms.updateMetaform(createdMetaform.id!!, createdMetaform.copy(
+    val createdScript2 = builder.metatavuAdmin.scripts.create(script)
```

Comment:
Is it intentional that all scripts are only tested for metatavu admins?

Reply:
Yes, creating, updating, and deleting is allowed only for Metatavu admins